### PR TITLE
Feature/kwix table component

### PR DESCRIPTION
**Pull request by VasiliyPolyakov**

### CODE REVIEW

File: src/components/table/table.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                        product.table_features.map((cell, index) => {
+                          if (index === 0) {
+                            return <th className="table__cell-first" key={index}>
+                              {this.getEvaluation(product.featured)}
+                              <div>{cell.feature_value}</div>
+                            </th>;
+                          } else if (cell.feature_key === 'price') {
+                            return (
+                              <td className="table__cell-last" key={index}>
+                                <AmazonLink customClass="amazon-link__table" link={product.amazon_link} />
```
Comment (mkozhukharenko) [inline src/components/table/table.js]: 'customClass' brakes component-driven approach.

correct approach:

```
<AmazonLink noBorder link={product.amazon_link} />
```

File: src/components/table/table.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import './table.scss';
+import AmazonLink from './../amazon-link/amazon-link';
+import BestValue from './../../img/silver-medal.svg'
```
Comment (mkozhukharenko) [inline src/components/table/table.js]: please, rename 'BestValue' to 'BestValueSvg'

File: src/templates/review-page.js [hunk 3]
```diff
@@ -62,6 +65,7 @@ export const ReviewPageTemplate = props => {
 
 const ReviewPage = ({ data }) => {
   const { frontmatter } = data.markdownRemark;
+  console.log('frontmatter', frontmatter)
```
Comment (mkozhukharenko) [inline src/templates/review-page.js]: redundant console.log

File: src/components/table/table.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+      default:
+        return null;
+    }
+  }
+
+  getEvaluation (evaluation) {
+    if (evaluation) {
+      return (
+        <div className="table__cell-rating">
+          <img className="table__cell-img" src={this.getIconRoute(evaluation)} alt=""/>
```
Comment (mkozhukharenko) [inline src/components/table/table.js]: 'alt' must not be empty

File: src/components/table/table.js (hunk 2 above)
Comment (VasiliyPolyakov) [reply src/components/table/table.js]: done

File: src/templates/review-page.js (hunk 3 above)
Comment (VasiliyPolyakov) [reply src/templates/review-page.js]: done

File: src/components/table/table.js (hunk 4 above)
Comment (VasiliyPolyakov) [reply src/components/table/table.js]: done

File: src/components/table/table.js (hunk 1 above)
Comment (VasiliyPolyakov) [reply src/components/table/table.js]: done

File: src/components/amazon-link/amazon-link.js [hunk 5]
```diff
@@ -4,11 +4,11 @@ import './amazon-link.scss';
 class AmazonLink extends Component {
 
   render() {
-    const {link} =  this.props;
+    const {link, text, isTableStyle} =  this.props;
 
     return (
-      <a className="amazon-link" target="_blank" rel="nofollow noopener" href={link}>
-        Buy on Amazon
+      <a className={isTableStyle ? 'amazon-link amazon-link__table' : 'amazon-link'} target="_blank" rel="nofollow noopener" href={link}>
```
Comment (mkozhukharenko) [inline src/components/amazon-link/amazon-link.js]: 'isTableStyle' is a modifier, so 'amazon-link__table' ->> 'amazon-link--is-table'

File: src/components/table/table.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+              }
+            </tr>
+            </thead>
+            <tbody className="table__body">
+              {
+                products.map((product, index) => {
+                  return (
+                    <tr className="table__row" key={index}>
+                      {
+                        product.table_features.map((cell, index) => {
```
Comment (mkozhukharenko) [inline src/components/table/table.js]: `table_features` could be null

build failed 
image

Comment (VasiliyPolyakov) [reply src/components/table/table.js]: done

File: src/components/amazon-link/amazon-link.js (hunk 5 above)
Comment (VasiliyPolyakov) [reply src/components/amazon-link/amazon-link.js]: done

**Pull request merged**